setup.py

```diff
@@ -6,5 +6,5 @@
       author='Ryan Jarvinen',
       author_email='ryanj@redhat.com',
       url='http://www.python.org/sigs/distutils-sig/',
-      install_requires=['Flask>=0.10.1', 'python-dateutil', 'httplib2'],
+      install_requires=['Flask>=0.10.1', 'python-dateutil', 'httplib2', 'oauthlib'],
```

Comment:
Is this replacing this file : https://github.com/proserve/github-issues-migrator/blob/master/github/oauth_client.py?